packages/@storefront/sayt/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [<release type>]
+### Changed
+- SF-1192: Only show sayt if search has finished fetching.
+  - While search is fetching, set sayt to inactive.
```

Comment:
Maybe add a brief explanation of the behaviour it fixed if you changed it to `fixed`.